src/adapters/rpc-server/services/upsert-friendship.ts

```diff
@@ -2,7 +2,7 @@ import { Action, FriendshipStatus, RpcServerContext, RPCServiceContext } from '.
 import {
   UpsertFriendshipPayload,
   UpsertFriendshipResponse
-} from '@dcl/protocol/out-ts/decentraland/social_service_v2/social_service.gen'
+} from '@dcl/protocol/out-ts/decentraland/social_service/v3/social_service_v3.gen'
```

Comment:
Doubt, is it okay to also have the v3 at the file name level and not only at route level?

Reply:
Good catch